### PR TITLE
Allow anyone to call claimFunds()

### PR DESCRIPTION
**Pull request by PumpkingWok**
Added new `address _recipient` parameter in `claimFunds()` and substituted `msg.sender` with it within this function to allow any address to reclaim it.

### CODE REVIEW

File: contracts/tests/round.ts [hunk 1]
```diff
@@ -503,6 +503,7 @@ describe('Funding Round', () => {
     const totalVotes = 100; // Math.sqrt(totalSpent)
     const recipientIndex = 3;
     const recipientClaimData = [
+      recipient.address, // recipient
```
Comment (xuhcc) [inline contracts/tests/round.ts:506]: A similar change is needed in `contracts/scripts/claim.ts`

Comment (PumpkingWok) [reply contracts/tests/round.ts:506]: Done

File: contracts/tests/round.ts [hunk 2]
```diff
@@ -549,6 +551,17 @@ describe('Funding Round', () => {
         .to.equal(expectedClaimableAmount);
     });
 
+    it('allows address different than recipient to claim allocated funds', async () => {
+      await token.transfer(fundingRound.address, matchingPoolSize);
+      await fundingRound.finalize(totalSpent, totalSpentSalt)
+
+      await expect(fundingRoundAsContributor.claimFunds(...recipientClaimData))
+        .to.emit(fundingRound, 'FundsClaimed')
+        .withArgs(contributor.address, expectedClaimableAmount);
```
Comment (xuhcc) [inline contracts/tests/round.ts:560]: It should emit event with recipient's address

Comment (PumpkingWok) [reply contracts/tests/round.ts]: Done

**Pull request merged**